src/Yesplan/EventManager.php

```diff
@@ -120,8 +119,12 @@ public function deleteOldEvents(): void
      */
     private function getDateTime(string $dateTimeString)
     {
-        $dateTime = DateTime::createFromFormat('Y-m-d\TG:i:se', $dateTimeString);
+        $dateTime = DateTime::createFromFormat('Y-m-d+', $dateTimeString);
```

Comment:
Do you actually mean to include `+` in the format?

Reply:
yes, it seems the format we get from yesplan is not always Y-m-d\TG:i:se, but can also be Y-m-d, or event worse just a timestamp.. The + as far as i can read from documentation makes the createFromFormat skip everything after +, instead of sending back a format error.

Reply:
If you know that the API will return date and time in one of three formats, then you should handle all of these and fail if the format does not match any of them. Are the datetime formats specified in the API documentation? I can only find https://manual.yesplan.be/en/developers/http-api-overview/#date-input-field that mentions dates and times.

Reply:
I dont - the API SAYS it will return a datetime, but it does not always, so I really don't know what will be returned..